drv/stm32xx-sys-api/src/h7.rs

```diff
@@ -158,12 +151,9 @@ pub enum Peripheral {
     Swp = periph(Group::Apb1H, 2),
     Crsen = periph(Group::Apb1H, 1),
 
-    #[cfg(feature = "h7b3")]
-    Dfsdm1 = periph(Group::Apb2, 30), // B3 differs from 43/47
-
     Hrtim = periph(Group::Apb2, 29), // 43/47 only
 
-    #[cfg(any(feature = "h743", feature = "h747", feature = "h757"))]
+    #[cfg(feature = "h743")]
     Dfsdm1 = periph(Group::Apb2, 28), // 43/47 differ from B3
 
     Sai3 = periph(Group::Apb2, 24), // 43/47 only
```

Comment:
**question:** There's a bunch of "B3 only" commented lines below: Are those now entirely unused?